sw/device/tests/example_test_from_flash.c

```diff
@@ -83,10 +84,88 @@ OTTF_DEFINE_TEST_CONFIG(.enable_concurrency = false,
  */
 // OT_SECTION(".non_volatile_scratch") uint32_t non_volatile_data[2];
 
+typedef struct node {
+  char name[2];
+  unsigned char type;
+  uint32_t param1;
+  uint32_t param2;
+} node_t;
+
 bool test_main(void) {
   /**
    * Place test code here.
    */
+  node_t n_arr[] = {
```

Comment:
@timothytrippel, I will revert these changes once the logic is working (I think it is, I need to verify it a bit more).